<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>1900</width>
      <height>720</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>82</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttons_layout">
     <item>
      <widget class="QPushButton" name="punto1">
       <property name="text">
        <string>Punto 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="punto2">
       <property name="text">
        <string>Punto 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="punto3">
       <property name="text">
        <string>Punto 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="punto4">
       <property name="text">
        <string>Punto 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="punto5">
       <property name="text">
        <string>punto 5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>20</y>
      <width>248</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="input_layout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="YC1_label">
         <property name="text">
          <string>Y primer cañon:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="YC1_slider">
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="YC1_lcd">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="96">
                 <red>164</red>
                 <green>166</green>
                 <blue>168</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>218</red>
                 <green>218</green>
                 <blue>218</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="XC2_label">
         <property name="text">
          <string>X segundo cañon:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="XC2_slider">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>1850</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="XC2_lcd">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="value" stdset="0">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="YC2_label">
         <property name="text">
          <string>Y segundo cañon:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSlider" name="YC2_slider">
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="YC2_lcd">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>181</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="display2">
     <item>
      <widget class="QLabel" name="shot_label2">
       <property name="text">
        <string>Disparo 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="angle_layout2">
       <item>
        <widget class="QLabel" name="angle_label2">
         <property name="text">
          <string>Angulo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="angle_lcd2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="velocity_layout2">
       <item>
        <widget class="QLabel" name="velocity_label2">
         <property name="text">
          <string>Velocidad inicial:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="velocity_lcd2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posx_layout2">
       <item>
        <widget class="QLabel" name="posx_label2">
         <property name="text">
          <string>Pos X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posx_lcd2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posy_layout2">
       <item>
        <widget class="QLabel" name="posy_label2">
         <property name="text">
          <string>Pos Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posy_lcd2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="time_layout2">
       <item>
        <widget class="QLabel" name="time_label2">
         <property name="text">
          <string>Tiempo de vuelo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="time_lcd2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>181</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="display3">
     <item>
      <widget class="QLabel" name="shot_label3">
       <property name="text">
        <string>Disparo 3:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="angle_layout3">
       <item>
        <widget class="QLabel" name="angle_label3">
         <property name="text">
          <string>Angulo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="angle_lcd3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="velocity_layout3">
       <item>
        <widget class="QLabel" name="velocity_label3">
         <property name="text">
          <string>Velocidad inicial:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="velocity_lcd3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posx_layout3">
       <item>
        <widget class="QLabel" name="posx_label3">
         <property name="text">
          <string>Pos X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posx_lcd3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posy_layout3">
       <item>
        <widget class="QLabel" name="posy_label3">
         <property name="text">
          <string>Pos Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posy_lcd3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="time_layout3">
       <item>
        <widget class="QLabel" name="time_label3">
         <property name="text">
          <string>Tiempo de vuelo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="time_lcd3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>191</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="display1">
     <item>
      <widget class="QLabel" name="shot_label1">
       <property name="text">
        <string>Disparo 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="angle_layout1">
       <item>
        <widget class="QLabel" name="angle_label1">
         <property name="text">
          <string>Angulo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="angle_lcd1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="velocity_layout1">
       <item>
        <widget class="QLabel" name="velocity_label1">
         <property name="text">
          <string>Velocidad inicial:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="velocity_lcd1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posx_layout1">
       <item>
        <widget class="QLabel" name="posx_label1">
         <property name="text">
          <string>Pos X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posx_lcd1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="posy_layout1">
       <item>
        <widget class="QLabel" name="posy_label1">
         <property name="text">
          <string>Pos Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="posy_lcd1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="time_layout1">
       <item>
        <widget class="QLabel" name="time_label1">
         <property name="text">
          <string>Tiempo de vuelo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="time_lcd1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
